scale-encode/src/lib.rs

```diff
@@ -17,8 +17,8 @@
 
 /*!
 `parity-scale-codec` provides an `Encode` trait which allows types to SCALE encode themselves based on their shape.
```

Comment:
We could; I think the only reason I don't do that is because the lib docs benefit from doc links (`[Foo]`) whereas in the README it's better to remove the []'s!

There's almost certainly a tool somewhere though which takes the lib docs and copies them into a README for you!

Reply:
It makes sense, the only crate I can think of might be: https://crates.io/crates/docify, something that substrate's been using for pallet docs